### DISCUSSION TITLE
希望扩充CQGetGroupInfo()方法返回的数据字段

### DISCUSSION THREAD

**Discussion by HuangShiJin1015** [Mrs4s/go-cqhttp#727] [Ideas]
CQGetGroupInfo()方法返回的数据字段只有群名称、群号、群人数、群最大人数四个字段，而Mirai中却包含很多有有用的字段。比如群简介、群创建时间、群头像等字段，希望能够扩充添加一下获取群信息返回的数据字段

**Comment by Ink-33** [comment]
#728
